sdk/core/azure-core/src/main/java/com/azure/core/implementation/JsonPatchOperation.java

```diff
@@ -104,31 +106,11 @@ public boolean equals(Object obj) {
 
     @Override
     public String toString() {
-        return toJson(new StringBuilder()).toString();
-    }
-
-    @Override
-    public StringBuilder toJson(StringBuilder stringBuilder) {
-        stringBuilder.append("{\"op\":\"")
-            .append(op.toString())
-            .append("\"");
-
-        if (from != null) {
-            stringBuilder.append(",\"from\":\"")
-                .append(from)
-                .append("\"");
-        }
-
-        stringBuilder.append(",\"path\":\"")
-            .append(path)
-            .append("\"");
-
-        if (value.isInitialized()) {
-            stringBuilder.append(",\"value\":")
-                .append(value.getValue());
-        }
+        AccessibleByteArrayOutputStream outputStream = new AccessibleByteArrayOutputStream();
+        JsonWriter writer = DefaultJsonWriter.toStream(outputStream);
```

Comment:
is `toStream` name right? It feels `fromStream` is more intuitive

Reply:
Yeah, it reads a bit odd to have the `to` prefix. `from` may be more appropriate.